src/pages/UserHome.jsx

```diff
@@ -81,27 +95,27 @@ export default function UserHome() {
           </Button>
         </Grid>
       </Grid>
-      <center>
+      <Box className={classes.centerIcon}>
+        <InboxIcon
+          style={{ fontSize: 200 }}
+          className={classes.addIconSpacing}
+        />
```

Comment:
I suggest making this a part of the `Box` below instead of a separate box. They're sort of belong together if you consider how:
1. They're all center aligned
2. They only appear when there are no scholarships
3. They're next to each other visually speaking.